cmd/dtab.go

```diff
@@ -108,12 +108,11 @@ var (
 	}
 
 	dtabUpdateCmd = &cobra.Command{
-		Use:     "update [name] [file]",
+		Use:     "update [name] [file] [version]",
```

Comment:
Also, I have a slight preference for using flags for optional arguments.  e.g. `update -v VERSION NAME FILE` or `update NAME FILE`.  I could go either way in this case, though.